cfme/fixtures/cli.py

```diff
@@ -0,0 +1,116 @@
+from utils.version import get_stream
+from cfme.test_framework.sprout.client import SproutClient
+from utils.appliance import current_appliance
+from utils.conf import cfme_data, credentials
+from utils.log import logger
+import pytest
+from wait_for import wait_for
+from cfme.test_framework.sprout.client import SproutException
+
+
+def is_dedicated_db_active(appliance2):
+    if appliance2.version < '5.7':
+        postgres = 94
+    else:
+        postgres = 95
+    return_code, output = appliance2.ssh_client.run_command(
+        "systemctl status rh-postgresql{}-postgresql.service | grep running".format(postgres))
+    return return_code == 0
+
+
+@pytest.fixture()
+def dedicated_db(appliance2, app_creds):
+    pwd = app_creds['password']
+    client = appliance2.ssh_client
+    channel = client.invoke_shell()
+    stdin = channel.makefile('wb')
+    stdin.write("ap \n 8 \n 1 \n 1 \n 1 \n y \n {} \n {} \n \n".format(pwd, pwd))
+    wait_for(is_dedicated_db_active, func_args=[appliance2])
+
+    return appliance2
+
+
+@pytest.yield_fixture(scope="function")
+def appliance():
```

Comment:
**REQUIRED:**  use here and below 
https://github.com/ManageIQ/integration_tests/blob/master/fixtures/appliance.py#L60